community-roles.md

```diff
@@ -1,49 +1,46 @@
-projects and initiatives, prioritizing official project roadmaps, and have the
-final say on project disputes.
-
-## Repository Manager
-The Repository Manager controls Github, NPM, and other Solid organization
-properties related to the storage, management, and distribution of official
-Solid projects. They are responsible for keeping these properties clean and
-well-organized on the community's behalf.
+The Community Leader defines the governing vision of the Solid project, and channels that vision into the projects and initiatives created to realise it. They are responsible for selecting core contributors, approving official projects and initiatives, prioritising official project roadmaps, and have the final say on project disagreements. The Community Leader has admin rights of Solid tools and can optionally attend the weekly recurring community support meeting.
 
 ## Community Manager
-Community Managers are responsible for community outreach, ensuring that
-community members are able to realize the full value of solid, and have what
-they need to contribute value back if they choose to do so.
+The Community Manager is responsible for community outreach to facilitate contributions and implementation of the governance of Solid. The Community Manager is responsible for merging pull requsets concerning the community repository and processing and implementing suggestions around Solid governance. The Community Manager has admin rights of Solid tools and must moderate the weekly recurring community support meeting. 
+
+## Repositories Manager
+The Repositories Manager controls Github, NPM, and other Solid organisation properties related to the storage, management, and distribution of official Solid projects. They are responsible for keeping these properties clean and well-organised on behalf of the Solid community. The Repositories Manager has admin rights of Solid tools must attend the weekly recurring community support meeting.
 
 ## Solid Core Contributor
-Solid Core Contributors work across projects on initiatives that enrich the
-Solid ecosystem as a whole. They must attend regular stand-ups and strategy
-meetings.
+Solid Core Contributors work across projects on initiatives that enrich the Solid ecosystem as a whole. Solid Core Contributors must attend the weekly recurring community support meeting.
 
 ## Project Core Contributor
-A Project Core Contributor is a key team member on an official Solid project.
-They may have issues assigned to them, review and merge pull requests, and must
-attend regular stand-ups and project strategy meetings.
-
-## Project Contributor
-A Project Contributor is anyone who contributes to an official Solid project.
-Contributions come in many forms, from coding and qa to experience design and
-technical content writing.
+A Project Core Contributor is a key team member on an official Solid project. Project Core Contributors must attend the weekly recurring community support meeting.
 
```

Comment:
We removed any specific responsibilities and replaced it with a short description that doesn't really specify anything concrete. I'm okay with merging project core contributor and project contributor because it's probably best to simplify right now, but the explanations need to be informative and helpful.

If you are a project core contributor, then you should be actively participating in that project and leading it efficiently and productively in the context of the overall Solid ecosystem. You should be able to have issues assigned, act on those issues, review and merge pull requests, and attend regular stand-ups and strategy meetings specific to that project. While I agree that there should be a regular community-wide dialogue across solid projects - I'm concerned that inviting everyone working on any solid community project to one weekly meeting could become unwieldy?